types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+import { Scenario } from "./models/scenario";
+
+export type {
+  Scenario
+}
```

Comment:
What is this file doing?

Reply:
this is requirement for TSOA so it can automatically generate routes and swagger files